spelling_wordlist.txt

```diff
@@ -159,3 +161,8 @@ verifier
 verifiers
 walkthrough
 whitespace
+prover_args
```

Comment:
this should be in code font (which is not spell checked) instead of added to the word list.